### PR TITLE
 Reformat scalameta/semanticdb

### PR DESCRIPTION
**Pull request by xeno-by**

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -3,18 +3,25 @@ align.openParenCallSite=false
 align.openParenDefnSite=false
 align.tokens = []
 assumeStandardLibraryStripMargin = true
+binPack.parentConstructors = false
```
Comment (olafurpg) [inline .scalafmt.conf]: I believe this is default

File: .scalafmt.conf [hunk 2]
```diff
@@ -3,18 +3,25 @@ align.openParenCallSite=false
...
+danglingParentheses = false
```
Comment (olafurpg) [inline .scalafmt.conf]: ditto

File: .scalafmt.conf [hunk 3]
```diff
@@ -3,18 +3,25 @@ align.openParenCallSite=false
...
+docstrings = ScalaDoc
```
Comment (olafurpg) [inline .scalafmt.conf]: ditto

File: .scalafmt.conf [hunk 4]
```diff
@@ -3,18 +3,25 @@ align.openParenCallSite=false
...
 maxColumn = 100
 optIn.annotationNewlines = true
+optIn.breakChainOnFirstMethodDot = true
```
Comment (olafurpg) [inline .scalafmt.conf]: ditto

File: .scalafmt.conf [hunk 5]
```diff
@@ -3,18 +3,25 @@ align.openParenCallSite=false
...
+optIn.selfAnnotationNewline = true
```
Comment (olafurpg) [inline .scalafmt.conf]: ditto

**Comment by xeno-by** [comment]
@olafurpg I just want to explicitly say that I'm interested in these particular values, because I have tried other values and didn't like the result. 

By settings these values explicitly, we are making sure that: 
  1) Potential changes in defaults in future versions of scalafmt will not affect us.
  2) Hypothetical future contributors that would like to change .scalafmt.conf will know that we're not just accidentally using the defaults, but are conscious in that choice.

**Pull request merged**